app/assets/json/translations/locale-es.json

```diff
@@ -442,7 +442,7 @@
         "footer": {
             "city_county_of_sf": "Ciudad y condado de San Francisco",
             "contact": "Contacto",
-            "dahlia_description": "<p class='c-iron'>DAHLIA: El Portal de Vivienda de San Francisco es un proyecto de la <br/><a class='a-white' href='{{mohcdUrl}}' target='_blank'>Oficina de Desarrollo Comunitario y de Viviendas del Alcalde</a></p><p class='t-small c-iron margin-top'>en colaboración con la<br/><a class='a-white' href='{{sfDeptUrl}}' target='_blank'>Oficina de Innovación Cívica del Alcalde (Mayor's Office of Civic Innovation)</a>del<br/><a class='a-white' href='{{mayorUrl}}' target='_blank'>Departamento de Tecnología de San Francisco (San Francisco Department of Technology)</a></p>",
+            "dahlia_description": "<p class='c-iron'>DAHLIA: El Portal de Vivienda de San Francisco es un proyecto de la <br/><a class='a-white' href='{{mohcdUrl}}' target='_blank'>Oficina de Desarrollo Comunitario y de Viviendas del Alcalde</a></p><p class='t-small c-iron margin-top'>en colaboración con la<br/><a class='a-white' href='{{sfdsUrl}}' target='_blank'>Oficina de Innovación Cívica del Alcalde (Mayor's Office of Civic Innovation)</a>del<br/><a class='a-white' href='{{mayorUrl}}' target='_blank'>Departamento de Tecnología de San Francisco (San Francisco Department of Technology)</a></p>",
```

Comment:
this might be a case where we could delete the translation because it's no longer accurate 🤔. Not a big deal, just a thought. 

We might be able to also update the translations without having to go to the vendor if we know how to translate "San Francisco Digital Services" in all of our supported languages.

Reply:
@akegan if we delete the translations, will it just use the English name until the new translations are added?

Reply:
It should just google translate the whole string. I asked in #topic-translations yesterday and it doesn't sound like we have translations for San Francisco Digital Services unfortunately so I think deleting it might be the best bet for now.